docs/frequently-asked-questions.rst

```diff
@@ -440,11 +224,8 @@ In this example::
         }
     }
 
-Can a contract function accept a two-dimensional array?
-=======================================================
+.. TODO: Does this mean it does work for internal?
```

Comment:
Not sure what this means.

Reply:
@chriseth Ah, this was an internal note to self, I will come back to this later and ask you the question.